### DISCUSSION TITLE
QPopupProxy different slot behavior than QPopupEdit (since Quasar 2). Unify behavior?

### DISCUSSION THREAD

**Discussion by 54mu3l** [quasarframework/quasar#13428] [Ideas / Proposals] [answered]
With Quasar 2 the behavior of QPopupEdit changed and we're supposed to use the default slot now.

Here are the two code examples from the docs upgrade guide (https://quasar.dev/start/upgrade-guide#qpopupedit):
```html
<!-- old way -->
<q-popup-edit
  content-class="bg-primary text-white"
  buttons
  color="white"
  v-model="myModel"
>
  <q-input
    type="textarea"
    dark
    color="white"
    v-model="myModel"
    autofocus
  />
</q-popup-edit>
```

```html
<q-popup-edit
  class="bg-primary text-white"
  buttons
  color="white"
  v-model="myModel"
  v-slot="scope"
>
  <q-input
    type="textarea"
    dark
    color="white"
    v-model="scope.value"
    autofocus
    @keyup.enter="scope.set"
  />
</q-popup-edit>
```

> Notice `v-slot="scope"` is applied directly on `<q-popup-edit>` and using `scope.value` instead of `myModel` for the inner `<q-input>` component

I noticed these changes are only for `<q-popup-edit>` and not for `<q-popup-proxy>` (As you can find in this example: https://quasar.dev/vue-components/popup-proxy#breakpoint).

**=> That means these similar components behave differently. Maybe it would make sense to unify their behavior.**
What do you think?

**Comment by pdanpdan** [comment, marked as answer]
They are not similar components. QPopupProxy does not use a model to pass to the component for changing. Also the slot can be used in Q1 too
